vulnerable_people_form/form_pages/check_your_answers.py

```diff
@@ -36,14 +36,16 @@ def get_check_your_answers():
     session["check_answers_page_seen"] = True
     return render_template_with_title(
         "check-your-answers.html",
-        previous_path="/basic-care-needs",
+        previous_path=dynamic_back_url(),
         summary_rows=get_summary_rows_from_form_answers(),
     )
 
 
 @form.route("/check-your-answers", methods=["POST"])
 def post_check_your_answers():
-    reference_number = persist_answers_from_session()
-    send_sms_and_email_notifications_if_applicable(reference_number)
+    session["nhs_sub"] = "a" * 36
```

Comment:
whats the reason for creating an nhs sub as 

"aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"

Reply:
Good spot - test code, since removed! 🙂